helm/templates/deployment.yaml

```diff
@@ -109,7 +125,7 @@ spec:
           secret:
             secretName: {{ .Values.caSecretName }}
```

Comment:
take ca secret according to the values changes (from global)

Reply:
done